src/content/blog/2024_toqito.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Toqito: Quantum Information Science Impact through Open Source"
+author: Vincent Russo and Purva Thakre
+day: 19
+month: 9
+year: 2024
+tags:
+    - python
+    - toqito
+    - microgrant
+    - qis
+---
+
+![](/images/toqito_logo.png)
+
+In early 2020, the humble [initial commit](https://github.com/vprusso/toqito/commit/a2f2449c4b9de81becff41cb25f1bca3fa180e8b) of the toqito project ([repo](https://github.com/vprusso/toqito)|[docs](https://toqito.readthedocs.io/en/latest/index.html)) made its way onto GitHub.
+
+![][/images/toqito_first_commit.png]
```

Comment:
Commenting to resolve this later: 

None of the images except for the logo show up in the preview. 

https://unitary-fund-prel-git-add-toqito-blog-post-unitary-fund.vercel.app/posts/2024_toqito